### PR TITLE
Comment on some things

### PR DESCRIPTION
**Pull request by sdwowk**
Add some documentation...

### CODE REVIEW

**Comment by linster** [comment]
Yep

File: QuestionAppTests/src/ca/ualberta/cs/cmput301f14t14/questionapp/test/AnswerTest.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ public void testAddAnswer() {
 	 */
 	public void testInvalidBody() {
 		// Test invalid body
-		Image image = new Image(null,null);
+		Image image = new Image(null,null, 0);
```
Comment (linster) [inline QuestionAppTests/src/ca/ualberta/cs/cmput301f14t14/questionapp/test/AnswerTest.java:56]: Wtf?

Comment (linster) [inline QuestionAppTests/src/ca/ualberta/cs/cmput301f14t14/questionapp/test/AnswerTest.java:56]: @sdwowk says there's a new constructor for image. Had no idea how to make it work. App tests were all broken. Added 0 for appeasement.

@lamorie , what does that argument do?

**Comment by lamorie** [comment]
Yes, there is a new constructor. Because the camera and the gallery work differently so I needed a simple way to identify the different images

**Pull request merged**